samples/subsys/fs/littlefs_blk/src/main.c

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright (c) 2021 Lukasz Majewski, DENX Software Engineering GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Sample which uses the filesystem API and SDHC driver */
+
+#include <zephyr.h>
+#include <device.h>
+#include <logging/log.h>
+#include <fs/fs.h>
+#include <fs/littlefs.h>
+#include <stdio.h>
+
+LOG_MODULE_REGISTER(main);
```

Comment:
LOG_MODULE_REGISTER(littlefs_blk_sample);

Reply:
I think that using "main" here is ok, many samples do so.

Reply:
Ok, I will keep this as is.